<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">::placeholder{
	margin-left: 10px;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>891</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #dee7fd;
border-radius: 15px;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>256</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Pacifico;
color: #2d2d2d;
font-size: 35px;</string>
    </property>
    <property name="text">
     <string>S7S Downloader</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Poppins, san-serif;
color: #4b4b4b;
font-size: 15px;</string>
    </property>
    <property name="text">
     <string>download any time any where</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 10px;
font-size: 15px;
font-family: poppins,sans-serif;
padding-left: 7px;</string>
    </property>
    <property name="placeholderText">
     <string comment="10">Download URL..</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 10px;
font-size: 15px;
font-family: poppins,sans-serif;
padding-left: 7px;</string>
    </property>
    <property name="placeholderText">
     <string comment="10">Save Location...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>200</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #3f38cb;
border-radius: 10px;
color: white;
font-family: poppins, sans-serif;
font-weight: Medium;
font-size: 12px;
}

QPushButton:hover{
	background-color:#2e2a9c ;
}</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>651</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
color:#a5a0ff ;
background-color: white;
border-radius: 10px;
font-size: 10px;
text-align: center;
font-family: poppins,sans-serif;
}
QProgressBar::chunk {
 background-color: #3f38cb; 
 border-radius: 10px;
}

</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #3f38cb;
border-radius: 10px;
color: white;
font-family: poppins, sans-serif;
font-weight: Medium;
font-size: 12px;
}

QPushButton:hover{
	background-color:#2e2a9c ;
}</string>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #3f38cb;
border-radius: 15px;
color: white;
font-family: poppins, sans-serif;
font-weight: Medium;
font-size: 12px;
}

QPushButton:hover{
	background-color:#2e2a9c ;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>320</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 10px;
font-size: 15px;
font-family: poppins,sans-serif;
padding-left: 7px;</string>
    </property>
    <property name="placeholderText">
     <string comment="10">File Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>320</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border-radius: 10px;
font-size: 15px;
font-family: poppins,sans-serif;
padding-left: 7px;</string>
    </property>
    <property name="placeholderText">
     <string comment="10">File Type : exe, png, jpg, etc....</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>down-icon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>down-icon.png</pixmap>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
